src/discussions/data/hooks.js

```diff
@@ -21,11 +21,12 @@ import { discussionsPath } from '../utils';
 import {
   selectAreThreadsFiltered, selectLearnersTabEnabled,
   selectModerationSettings,
-  selectPostThreadCount,
+  selectPostThreadCount, selectThreadCallStatus,
```

Comment:
Test cases are failing and the selector is not found in the relative file. Please attach the before, and after videos.